### ISSUE TITLE
[@firebase/rules-unit-testing] InitializeTestApp fails - storage is not a function

### ISSUE THREAD

**Issue by dennemark** [firebase/firebase-js-sdk#4944]
### Describe your environment

  * Operating System version: Windows 10
  * Browser version: Chrome 90
  * Firebase SDK version: @firebase/rules-unit-testing v. 1.3.2 + newest emulators
  * Firebase Product: storage

### Describe the problem

When trying to test storage rules, I cannot initialize the test app, supplying the storageBucket parameter. I get the following error: `TypeError: app.storage is not a function`

This should be the minimal code example:

```
import * as fbTesting from "@firebase/rules-unit-testing";

const app = fbTesting.initializeTestApp({
        projectId: MY_PROJECT_ID, 
        storageBucket: MY_STORAGE_BUCKET,
        auth: MY_AUTH})
```
If I use the admin app instead `fbTesting.initializeAdminApp(...)`, I can use the storageBucket parameter and write to the emulator.

I read in a few posts, that it is necessary to import `firebase/storage`, but this happens here: https://github.com/firebase/firebase-js-sdk/blob/b49345d31cdd3dfd42d65768156818dc09c7fa61/packages/rules-unit-testing/src/api/index.ts#L21

**Comment by dennemark** [comment]
It seems, my react app using reactfire is running into a similar issue, complaining about app.storage is not a function.

**Comment by looptheloop88** [comment]
Hi @dennemark, I wasn't able to reproduce the error that you're getting. Would you mind sharing a [minimal, but complete](https://stackoverflow.com/help/minimal-reproducible-example) sample project that I can run locally to replicate the issue?

```
import * as firebase from "@firebase/rules-unit-testing";

firebase.initializeTestApp({ projectId: PROJECT_ID, storageBucket: MY_STORAGE_BUCKET, auth: MY_AUTH }).storage();
```

I'm using the following dependencies:
```
"@firebase/rules-unit-testing": "^1.3.2"
"firebase-tools": "^9.12.0"
```

**Comment by dennemark** [comment]
Hi,

thanks for your effort!

I think I just found the issue. It actually is related to yarn workspaces and how it resolves node_modules. I do not fully understand my solution, but what I did was upgrading to yarn 2 (probably not necessary) and deleting yarn.lock files in my subrepositories. Afterwards I ran yarn install and now it seems to work again.

I will close this issue as it seems to be resolved!
